docker-compose.yml

```diff
@@ -60,15 +60,15 @@ services:
       - ./:/code
     environment:
       DATABASE_HOST: postgres
-      BROKER_URL: amqp://guest:guest@rabbitmq:5672/
-      SHARE_API_URL: http://web:8000/
-      ELASTICSEARCH_URL: http://elasticsearch:9200/
+      BROKER_URL: amqp://guest:guest@rabbitmq:35672/
+      SHARE_API_URL: http://web:38000/
+      ELASTICSEARCH_URL: http://elasticsearch:39200/
 
   web:
     build: .
-    command: python manage.py runserver --noreload 0.0.0.0:8000
+    command: python manage.py runserver --noreload 0.0.0.0:38000
```

Comment:
Does this work? You've told the server in the container to run on port 38000 but you're forwarding the host's port 8000 in the container still.

https://docs.docker.com/compose/compose-file/#ports